src/third-party/text/2.64.0/remarkable.min.js

```diff
@@ -0,0 +1,4 @@
+/*! remarkable 1.7.1 https://github.com/jonschlinkert/remarkable @license MIT */
```

Comment:
Looks like this is the only asset we don't have in a non-minified state. Does that matter?